phpBB/includes/functions_content.php

```diff
@@ -362,26 +362,28 @@ function get_context($text, $words, $length = 400)
 			$start = $end = 0;
 			foreach ($word_indizes as $indize => $word)
```

Comment:
`indize` is more a German word, may I ask the reason why?

Reply:
I kept some variable names from the original code while rewriting it

Reply:
I see :+1: